### DISCUSSION TITLE
undeploy without npm credentials

### DISCUSSION THREAD

**Discussion by michela** [mozilla/hubs#2976] [General Questions & Discussion] [answered]
We've managed to break the signin flow with a custom deploy. Is there a way to undeploy manually without the npm process? e.g. via AWS

**Comment by robinkwilson** [comment]
Sorry, I can't test this right now, but can you check the AWS Parameter store for values that seem to point to a custom client? 

Double check it's the same region as your CloudFormation stack. You may be able to delete the value to revert back to defaults. 

Please let me know if this works!

**Reply by johnshaughnessy** [reply]
I'm marking this as the answer for now to remove it from our unanswered stack, but if the above solution does not work, feel free to un-mark it. @michela @robin-k-wilson

**Comment by robinkwilson** [comment, marked as answer]
From (https://github.com/mozilla/hubs/discussions/3657) 

To remove the custom client without logging in with `npm run login`:

1. Delete the Parameter Store config
     1. Go to AWS Console > Systems Manager > Parameter Store
     1. Delete config called `ita/<your HC stack name>/hubs/deploy/type`
1. Force a config refresh by updating the stack
     1. Go to AWS Console > Cloudformation > Stacks - _check your region! if you see 0 stacks_
     1. Update your EC2 server size from `t3.small` -> `t3.medium` - _you'll change this back after!!!_
1. Undeployed custom client! Check your HC landing page!
1. Change the EC2 server size back to the old config 
     1. Update the stack from `t3.medium` -> `t3.small`
